environment.yml

```diff
@@ -18,6 +15,9 @@ dependencies:
   - tqdm
   - urllib3
   - pip:
+    - hvac
```

Comment:
this packages exist in conda, we should add them to the dependencies root, instead of  in `pip`